src/main/java/io/burt/athena/AthenaDriverInfo.java

```diff
@@ -0,0 +1,42 @@
+package io.burt.athena;
+
+import java.io.IOException;
+import java.util.Properties;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class AthenaDriverInfo {
+    private static final String driverVersion;
+    private static final int driverMajorVersion;
+    private static final int driverMinorVersion;
+
+    static {
+        final Properties properties = new Properties();
+        try {
+            properties.load(AthenaDatabaseMetaData.class.getClassLoader().getResourceAsStream("project.properties"));
+        } catch (IOException e) {
+            throw new RuntimeException("Could not load driver version: " + e.getMessage(), e);
+        }
+        driverVersion = properties.getProperty("project.version");
+        final Pattern versionComponentsPattern = Pattern.compile("(\\d+)\\.(\\d+)\\.(\\d+)(?:-\\w+)?");
+        final Matcher matcher = versionComponentsPattern.matcher(driverVersion);
+        if (matcher.matches()) {
+            driverMajorVersion = Integer.parseInt(matcher.group(1));
+            driverMinorVersion = Integer.parseInt(matcher.group(2));
+        } else {
+            throw new RuntimeException("Could not parse driver version: " + driverVersion);
```

Comment:
What happens if you throw a runtime exception from a static block?

Reply:
You will get the error when you try to check the version:
```
Exception in thread "main" java.lang.ExceptionInInitializerError
	at io.burt.athena.AthenaDatabaseMetaData.getDriverVersion(AthenaDatabaseMetaData.java:42)
	at io.burt.athena.Test.main(Test.java:16)
Caused by: java.lang.RuntimeException: Could not parse driver version: 0.3a.0-SNAPSHOT
	at io.burt.athena.AthenaDriverInfo.<clinit>(AthenaDriverInfo.java:27)
	... 2 more
```